README.md

```diff
@@ -121,10 +121,10 @@ Join a channel:
 /join [channel]
 ```
 
-Start a direct chat with someone:
+Start a direct chat with someone or multiple users:
 ```
-/query [username]
-/slack talk [username]
+/query [username] ([username2] [username3]...)
+/slack talk [username] ([username2] [username3]...)
```

Comment:
And a minor thing. `[...]` denotes an optional value, while `<...>` denotes a variable. So this should be:
`/slack talk <username> [<username2> [<username3>...]]`

(I see that this wasn't correct for this command previously either.)

Reply:
Sounds good! Yeah, wasn't sure how to format this given the previous conventions.